client/src/api/tickets.ts

```diff
@@ -0,0 +1,38 @@
+import {
+    IAddNewCommentActionPayload,
+    IChangeTicketStatusPayload,
+    ICreateTicketActionPayload,
+    IQueryParamsPayload,
+    ISetTicketsPayload
+} from '@redux/types/tickets';
+import { api } from '@services/api';
+
+export const getAllTicketsApi = (payload: IQueryParamsPayload): Promise<ISetTicketsPayload[]> => {
+    return api.get('/tickets', {
+        params: payload
+    });
```

Comment:
please change the property name from 'payload' to 'params' for simplify object creating
`{ params }`